src/Actions/ThemeSupports.php

```diff
@@ -53,6 +53,9 @@ public function bcgov_block_theme(): void {
 
 		// Add support for responsive embedded content.
 		add_theme_support( 'responsive-embeds' );
+
+		// Adds excerpt support to pages.
+		add_post_type_support( 'page', 'excerpt' );
```

Comment:
looks good. 

maybe in future prs, try doing only one thing.

That will make it easier for others to review your work, because they will know what to look for.

Reply:
not sure what you mean, this was really simple, added post support, and updated links in changelog

Reply:
Of course, I should have been more clear.

I meant the Url change in the changelog is the second thing in this PR. 

Sorry for the confusion.